docs/GoTopButton.md

```diff
@@ -0,0 +1,39 @@
+📢 Use this project, [contribute](https://github.com/vtex-apps/store-components) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
+
+# Go Top Button
+
+`GoTopButton` is a component of VTEX that when clicked, goes back to the top of the page. This Component can be imported and used by any VTEX app.
+
+## Configuration
+
+1. Import the `vtex.store-component` app to your theme's dependencies in the manifest.json;
+
+```json
+ "dependencies: {
+    "vtex.store-components": "3.x"
+  }
+```
+
+2. Add the `go-top-button` block into your page. Example:
```

Comment:
2. Add the `go-top-button` block into a store template of your choosing. In the example, it will be added to the home page: